pkg/instrumentation/helper.go

```diff
@@ -123,6 +124,22 @@ func isInstrWithoutContainers(inst instrumentationWithContainers) int {
 	return 0
 }
 
+func instrVolume(volume corev1.Volume, name string, volumeSizeLimit *resource.Quantity) (corev1.Volume, error) {
```

Comment:
Could you add a docstring for this function?

Reply:
Added docstring